src/ol/expr/cpu.js

```diff
@@ -105,7 +105,8 @@ function compileExpression(expression, context) {
       return compileAssertionExpression(expression, context);
     }
     case Ops.Get:
-    case Ops.Var: {
+    case Ops.Var:
+    case Ops.Concat: {
```

Comment:
I think you can put the implementation of the expression here instead of in `compileAccessorExpression`, because `concat` cannot really be considered an accessor expression; I guess eventually we might have a `compileStringExpression` function if we have more than just concat :)

Reply:
Alright, I can move it. I personally can't see many more string expressions being a thing, but I could be missing some use cases that people have.

Reply:
I was thinking of uppercase, lowercase, capitalize, substring etc.